crates/uv/src/main.rs

```diff
@@ -148,7 +153,9 @@ async fn run() -> Result<ExitStatus> {
     };
 
     // Configure the `warn!` macros, which control user-facing warnings in the CLI.
-    if !globals.quiet {
+    if globals.quiet {
+        uv_warnings::disable();
+    } else {
         uv_warnings::enable();
     }
```

Comment:
Reflect workspace quiet flag after workspace loading.
Added new uv_warnings::disable() for this process.